float64 drone_lat
float64 drone_lon
float64 drone_hmsl
